packages/web3-shared/base/src/specs/index.ts

```diff
@@ -1067,6 +1078,14 @@ export interface Hub<ChainId, SchemaType, GasOption, Web3HubOptions = HubOptions
     cancelOrder?: (/** TODO: add parameters */) => Promise<void>
 }
 
+export interface Storage {
+    has?(key: string): Promise<boolean>
+    get<T>(key: string): Promise<T | NextIDStorageInfo<T> | undefined>
```

Comment:
I understand that this is a strange type of return value, but the data structure stored in `NextId` is multiple Proofs, not a Key-value structure. If we need the developer to notice and use the type correctly, we need to declare a new `NextIDStorage` or declare a `getNextIdData`. But I think this would break the consistency of `Storage`.